Cargo.toml

```diff
@@ -23,16 +23,23 @@ log = "0.4"
 rand = "0.4"
 serde = "1.0"
 serde_derive = "1.0"
+sha2 = "^0.7"
+hkdf = "^0.5"
+ring = "0.13"
 signatory = "0.6"
 simplelog = "0.5"
 toml = "0.4"
 yubihsm = { version = "0.14", optional = true }
-prost = { git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
-prost-derive = { git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
 serde_json = "1.0"
 hex = "0.3.2"
 bytes = "0.4"
 chrono = "0.4.2"
+byteorder = "^1.2"
+lazy_static = "1.1.0"
+x25519-dalek = "^0.1"
```

Comment:
It looks like this is an old version and that's what's causing it to pull in an old version of `subtle`. Can you bump this to `0.3`? (the `^` is also unnecessary as Cargo is semver-by-default)

Reply:
Thanks, updated `x25519-dalek` to `0.3 and `rand` to `0.5`. This still fails on stable.